### ISSUE TITLE
recently used documents; pin recently used documents

### ISSUE THREAD

**Issue by Golddouble** [M7S/dockbarx#153]
I have read, that dockbarx does also support recently used documents and also has the possibility to pin recently used documents like Windows 7 has. This was one of the reason, why I was so happy finally have found dockbarx.

But this two functions are not there. I have not found them. See here:

![Win7 Taskl](https://user-images.githubusercontent.com/13977359/97077944-3faf9680-15e8-11eb-8a3a-bb1c84598a02.png)
![Win7 Tasklist](https://user-images.githubusercontent.com/13977359/97077923-fbbc9180-15e7-11eb-822d-06821717448f.png)

(MX Linux (Debian 10/Xfce)

Thank you.

**Comment by xuzhen** [comment]
install zeitgeist to enable this optional function.
```sudo apt install zeitgeist gir1.2-zeitgeist-2.0```

**Comment by Golddouble** [comment]
Thank you.
My error was, that I had installed only "zeitgeist", but not "gir1.2-zeitgeist-2.0"

I have now "recently used documents":
![Win7 Taskbar letzte](https://user-images.githubusercontent.com/13977359/97082969-924f7980-160d-11eb-9a97-9723a4cbb4db.png)

But I have not found a "pin-function" for the documents in "recently used". Do we have a pin function in "dockbarx" for ducuments like Windows 7 has?

Thank you.

**Comment by xuzhen** [comment]
You can't pin recently used documents directly. But creating a custom launcher for those documents is possible.

**Comment by Golddouble** [comment]
Not sure if I understand right. 
Do you suggest to create a separate icon on the dockbarx taskbar only for one document. One starter for one document?

**Comment by xuzhen** [comment]
Yes. There is no pin function for recently used documents currently, and I don't think someone would add such features in the near future. If you want to pin some documents, you have to create a separate launcher for each of them.

**Comment by Golddouble** [comment]
Thank you.
